src/useCustomChange.ts

```diff
@@ -0,0 +1,10 @@
+import {useState} from "react";
+
+const useCustomChange=()=>{
+    const[isChange,setChange]=useState(false);
```

Comment:
```suggestion
    const[isChange,setIsChange]=useState(false);
```
useState를 사용하실 때는 set 함수의 이름을 상태변수명 그대로 활용하시는게 좋습니다~